keyboards/lets_split/keymaps/suzuken/rules.mk

```diff
@@ -0,0 +1,3 @@
+ifndef QUANTUM_DIR
```

Comment:
This file can be removed.

Specifically, the contents can be removed completely, as the ifdef block is a holdover from the old make system, and that leaves an empty file.